src/customers.ts

```diff
@@ -151,3 +156,41 @@ export async function getUsedKubernetesDistributions(vendorPortalApi: VendorPort
 
   return kubernetesDistributions;
 }
+
+export async function listCustomersByName(vendorPortalApi: VendorPortalApi, appSlug: string, customerName: string): Promise<CustomerSummary[]> {
+  const http = await vendorPortalApi.client();
+
+  // 1. get the app
+  const app = await getApplicationDetails(vendorPortalApi, appSlug);
+
+  // 2. list customers filtered by name
+  const listCustomersUri = `${vendorPortalApi.endpoint}/app/${app.id}/customers?name=${encodeURIComponent(customerName)}`;
```

Comment:
I think using https://replicated-vendor-api.readme.io/reference/searchteamcustomers is intended for search, this list customer endpoints fetched all customers from database and only filter after the database query. Also it has pagination that we have yet handled